.changeset/common-signs-grin.md

```diff
@@ -0,0 +1,5 @@
+---
+'sv': patch
+---
+
+fix: Project steps alignment
```

Comment:
At least in SvelteKit we usually start changelog entries with a lowercase. Not sure if Manuel has been doing the same here or not
```suggestion
fix: align project steps
```

Reply:
I do